build.assets/Makefile

```diff
@@ -263,6 +264,10 @@ clean:
 .PHONY: publish-telekube
 publish-telekube: build
 	TELEPORT_REPOTAG=$(TELEKUBE_TSH_TAG) $(MAKE) build-tsh
+	if [ -z "$(TELE_KEY)" ] || [ -z "$(DISTRIBUTION_OPSCENTER)" ]; then \
+	   echo "TELE_KEY or DISTRIBUTION_OPSCENTER are not set"; exit 1; \
+	fi;
+	tele login --ops $(DISTRIBUTION_OPSCENTER) --key=$(TELE_KEY) $(TELEKUBE_PUBLISH_FLAGS)
```

Comment:
This will use `tele` from path which might not be what we want. I'm also not sure tele login should be a part of the publish target. I would remove these changes for now and only kept the change to remove "tele logout" from robobest.

Reply:
Also, we do tele login elsewhere as part of the publish target:

https://github.com/gravitational/gravity/blob/899d5f528c04e52c8e2eecf827d68f09be60af1a/Makefile#L453-L460

I won't mess with that in the backports but it is something to keep in mind if "tele login [shouldn't be a part of the publish target".

Reply:
@wadells Yeah, good points and I agree with them, my main driver was just to follow our process of "fix regression asap, follow up with long-term fix after". The publishing change is good, but needs careful testing - once this is merged, you can open a pull request with those changes again (provided you've tested them and confirmed publishing works on all branches).